gulpfile.babel.js

```diff
@@ -0,0 +1,29 @@
+import gulp from 'gulp';
```

Comment:
It's part of the project specification to use gulp to serve the app and run tasks